### PR TITLE
Miscellaneous mission improvements 2

### PR DESCRIPTION
**Pull request by Smittytron**
More minor fixes and additions from my mission to do list.
- Removes unneeded fake building  overrides from Allies 02 and Suvival01
- Links all missions with campaign-global.lua which will let us start migrating common functions like `IdleHunt` over.
- Changes Moneycrate to the correct `wcrate`. (wcrate uses the wrong palette and should be green, but that's an issue for later)
- Adds infantry production to Soviet02a and 02b.
- Fixes whitespace issues in mission browser.

### CODE REVIEW

File: mods/ra/rules/campaign-rules.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@ MONEYCRATE:
 	GiveCashCrateAction:
 		Amount: 2000
 	RenderSprites:
-		Image: scrate
+		Image: wcrate
```
Comment (pchote) [inline mods/ra/rules/campaign-rules.yaml]: `wcrate` is already the default defined in rules/misc.yaml, so we can instead just delete this override.

It turns out that wcrate.shp is actually using player remap, and by eye it looks like it's using France's colour range. We can therefore fix this by changing the definition in rules/misc.yaml to:

```
RenderSprites:
	Image: wcrate
	Palette: wcrate
```

and adding a new palette definition in rules/palettes.yaml:
```
IndexedPalette@wcrate:
	Name: wcrate
	BasePalette: player
	Index: 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95
	ReplaceIndex: 224, 224, 225, 225, 226, 184, 185, 186, 187, 188, 188, 189, 190, 190, 191, 191
```

**Comment by pchote** [comment]
LGTM otherwise 👍

File: mods/ra/rules/campaign-rules.yaml (hunk 1 above)
Comment (Smittytron) [reply mods/ra/rules/campaign-rules.yaml]: Thanks! I'll tackle that in a separate PR.

**Comment by Smittytron** [comment]
Updated

**Review by pchote — APPROVED** [review]

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/5ca720abd0bb883c227cc0c7bea3f78507f56067...cabfa1891b3ac9652ddfc476d9680c47ff176008

**Pull request merged**